### ISSUE TITLE
Аddon works a little wrong

### ISSUE THREAD

**Issue by Vitoshka** [GritsUI/ManaMinder#8]
I noticed one minor feature of the add-on, which works incorrectly. Addon periodically shows the icon of consumables, then you can use it even if I did not use it.

Also, the add-on interacts oddly with addon [ItemRack](https://github.com/Linae-Kronos/ItemRack), when I change my clothes into another outfit, then the icon of the consumables appears again, that I can use it even if I did not use it.

**Comment by GritsUI** [comment]
I may misunderstand, but are you saying you see the alert of a consumable being ready pop up multiple times without having used it?

Currently, an alert is shown when the top consumable becomes available, if it was previously in a not available state. So it's possible to get multiple alerts if your mana drops to a point where you can use a potion, then it regens to a point where you have too much mana to use the potion, then you drop back down to a point where you can use it again.

It's possible something similar is happening when you change your gear if you are equipping/unequipping an item that restores mana. Could that be the case? Or are you seeing an alert unrelated to the gear you're changing?

I've been thinking about adding something to limit the frequency of alerts in situations like that. It can be annoying if your mana is hovering around the ready point for a consumable.

**Comment by Vitoshka** [comment]
Yes, just as you described, can you add the option of disabling and enabling the icon for displaying this function based on the value of the mana?
 
Is it also possible to display a double icon for the willingness of the dark rune and demonic rune? Or add options which rune I use at a given time, if only one icon is displayed.

**Comment by GritsUI** [comment]
Ok, I'll have a look at improving that behaviour soon.

On the dark/demonic runes, you can choose which is used first by moving it up the "Tracked" list. When any consumables are combined like that, it uses the highest priority one. I'm planning on providing a bit more help text on hover to some options that should make that more clear.

**Comment by GritsUI** [comment]
I've added a "Repeat Delay" option to the alerts tab that should improve this situation. It's set to 5 seconds by default, which means a consumable needs to be "not ready" for 5 seconds before it will fire a new alert on becoming available. I raided with it last night and this felt about right; I didn't notice any unwanted alerts.

Available in [Version 1.4](https://github.com/GritsUI/ManaMinder/releases/tag/v1.4) or the latest from master.
